core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -279,6 +285,23 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     case RescheduleJob =>
       freePool = freePool - sender()
       busyPool = busyPool - sender()
+    case prewarmConfigList: PreWarmConfigList =>
+      laststPrewarmConfig = prewarmConfigList.list
```

Comment:
Add a inner variable `laststPrewarmConfig`, just make it pointed to the passed config runtime.
Because in this patch: https://github.com/apache/openwhisk/pull/4698
when received `ContainerRemoved`(may be prewarm container crashed), it will backfill the prewarm.